src/lib/Scenes/Artwork/Artwork.tsx

```diff
@@ -34,11 +36,13 @@ export class Artwork extends React.Component<Props> {
           </Flex>
           <Separator />
           <Join separator={<Spacer my={2} />}>
+            <ReadMore content={artwork.artist.biography_blurb.text} maxChars={140} />
```

Comment:
This already exists in the `AboutArtist` component, I'm not sure why it got added here but we should remove!

Looks like we probably don't need the query for these fields below either. That likely got kept in there when I moved that component! 😞